cachelib/navy/testing/MockDevice.h

```diff
@@ -44,6 +44,14 @@ class MockDevice : public Device {
   MOCK_METHOD3(readImpl, bool(uint64_t, uint32_t, void*));
   MOCK_METHOD3(writeImpl, bool(uint64_t, uint32_t, const void*));
   MOCK_METHOD0(flushImpl, void());
+  MOCK_METHOD0(allocatePlacementHandle, int());
+
+  virtual bool writeImpl(uint64_t offset,
```

Comment:
remove `virtual` here and add `override` instead. i.e., `bool writeImpl(...) override {`

Reply:
This results in compilation error (due to gtest restrictions on default argument - int handle - I guess).
How about adding a line ?
MOCK_METHOD4(writeImpl, bool(uint64_t, uint32_t, const void*, int));